### PR TITLE
test: Add missing test coverage for auto_image, auto_mathjax, auto_wiktionary

### PR DESCRIPTION
**Pull request by ryusoh**
Adds missing unit tests for `auto_image`, `auto_mathjax`, and `auto_wiktionary` to handle edge cases and exception branches that were previously unexercised. Coverage for the main `__init__.py` files of these modules is now at 100%. All pre-commit formatting and check steps have been completed.

---
*PR created automatically by Jules for task [16969609409061493302](https://jules.google.com/task/16969609409061493302) started by @ryusoh*

### CODE REVIEW

File: plan.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+1. **Explore Auto Image Add-on (Completed)**
```
Comment (ryusoh) [inline plan.md]: clean up

File: run_pytest_cov.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (ryusoh) [inline run_pytest_cov.sh]: clean up

File: test_auto_image_addition.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import pytest
```
Comment (ryusoh) [inline test_auto_image_addition.py]: why it is in the root dir
